sample/F8/sessions

```diff
@@ -0,0 +1,1470 @@
+         "name":"Hall G"
+      }
+   },
+   {
+      "short_abstract":"",
+      "id":7,
+      "subtitle":"litho-a-declarative-framework-for-efficient-UIs",
+      "title":"Litho: A Declarative Framework for Efficient UIs",
+      "long_abstract":"Components for Android allows developers to implement complex, efficient RecyclerViews through a simple declarative API. In this session we’ll discuss the function and features of this powerful framework.",
+      "end_time":"2017-04-18T13:20:00",
+      "microlocation":{
+         "id":6,
+         "name":"Hall F"
+      },
+      "start_time":"2017-04-18T12:30:00",
+      "comments":"",
+      "language":"",
+      "slides":"",
+      "audio":"",
+      "video":"",
+      "signup_url":"",
+      "state":"accepted",
+      "session_type":{
+         "id":6,
+         "name":"50 minutes session",
+         "length":"00.50"
+      },
+      "track":{
+         "id":3,
+         "name":"Engineering & Open Source",
+         "color":"#E8782C"
+      },
+      "speakers":[
+         {
+            "city":"",
+            "heard_from":"",
+            "icon":"",
+            "long_biography":"",
+            "small":"",
+            "photo":"",
+            "thumbnail":"",
+            "short_biography":"",
+            "speaking_experience":"",
+            "sponsorship_required":"",
+            "organisation":"Facebook",
+            "id":4,
+            "name":"Lucas Rocha "
+         },
+         {
+            "organisation":"Facebook",
+            "id":5,
+            "name":"Emil Sjölander"
+         }
+      ]
+   },
+   {
+      "short_abstract":"",
+      "id":8,
+      "subtitle":"webvr-enabling-developers-to-build-engaging-cross-platform-vr-content",
+      "title":"WebVR: Enabling Developers to Build Engaging Cross-Platform VR Content",
+      "long_abstract":"Web-based frameworks such as React VR are making it easier than ever to create VR content that can be shared with a huge audience across VR, mobile, and the web. Coupled with the new web browser that Oculus is building, more of the web will become immersive over the coming years. Join us to learn what you can build with WebVR and how this new technology can help enhance your existing content.",
+      "start_time":"2017-04-18T12:30:00",
+      "end_time":"2017-04-18T13:20:00",
+      "microlocation":{
+         "id":2,
+         "name":"Hall LL"
+      },
+      "comments":"",
+      "language":"",
+      "slides":"",
+      "audio":"",
+      "video":"",
+      "signup_url":"",
+      "state":"accepted",
+      "session_type":{
+         "id":6,
+         "name":"50 minutes session",
+         "length":"00.50"
+      },
+      "track":{
+         "id":5,
+         "name":"Virtual Reality",
+         "color":"#B21212"
+      },
+      "speakers":[
+         {
+            "city":"",
+            "heard_from":"",
+            "icon":"",
+            "long_biography":"",
+            "small":"",
+            "photo":"",
```

Comment:
add speaker images. if not available on site please google them. do this for all